modules/installation-gcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,77 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-config_{context}"]
+= Sample customized `install-config.yaml` file for shared VPC installation
```

Comment:
We don't put objects/anything in backticks in the heading: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings